docs/content/release-notes/0.2.0.md

```diff
@@ -0,0 +1,44 @@
+---
+title: "0.2.0"
```

Comment:
looks like we need a "slug" field here too since it has a dot. `slug: "0-2-0"` will do the trick